src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/PoromechanicsKernels.cmake

```diff
@@ -21,7 +21,10 @@ set( porousSolidDispatch PorousSolid<DruckerPragerExtended>
                          PorousSolid<ElasticOrthotropic>
                          PorousSolid<DamageSpectral<ElasticIsotropic>>
                          PorousSolid<DamageVolDev<ElasticIsotropic>>
-                         PorousSolid<Damage<ElasticIsotropic>> )
+                         PorousSolid<Damage<ElasticIsotropic>>
+                         PorousSolid<DuvautLionsSolid<DruckerPrager>>
+                         PorousSolid<DuvautLionsSolid<DruckerPragerExtended>>
+                         PorousSolid<DuvautLionsSolid<ModifiedCamClay>> )
```

Comment:
Before merging, we should find a way to pass the CI test by reducing the number of options here: https://fcmaelstrom.slack.com/archives/C03U0RPR5K2/p1678918573857919?thread_ts=1678914869.324949&cid=C03U0RPR5K2

Reply:
Right, this becomes an obvious hurdle on Travis build

Reply:
@Jian, let's just comment out these instances for the moment (anything with DuvautLionSolid) until we resolve the build time issues.